src/app/lib/presentation/diagnostic/environment_picker_widget.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:app/business/environment/environment_manager.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+
+/// Widget that handles environment switching.
+class EnvironmentPickerWidget extends StatefulWidget {
+  const EnvironmentPickerWidget({super.key});
+
+  @override
+  State<StatefulWidget> createState() {
+    return _EnvironmentPickerWidgetState();
+  }
+}
+
+class _EnvironmentPickerWidgetState extends State<EnvironmentPickerWidget> {
+  final _environmentManager = GetIt.I<EnvironmentManager>();
+
+  bool restartRequired = false;
+  late String selectedEnviroment;
```

Comment:
Can't we use `_environmentManager.next` here as default value?

Reply:
hum I thought that's what I did...

Reply:
If I recall, you can't declare properties by referencing other properties.

Reply:
You are probably right

Reply:
![image](https://github.com/nventive/FlutterApplicationTemplate/assets/17461593/e2f28f15-8aca-4123-85fc-f1de95680b3c)